include/executor/executor.h

```diff
@@ -137,6 +138,9 @@ class Executor {
     if (Stat) {
       Stat->setCostLimit(Conf.getStatisticsConfigure().getCostLimit());
     }
+    if (Conf.getRuntimeConfigure().isEnableCoredump()) {
+      std::cout<<"Coredump has been enabled in executor";
```

Comment:
please config your editor so these sort of format problem can be automatically eliminated.

Reply:
Sure. Seems like clang-format wasn't running correctly on CLion. Has been fixed on my end now. Since this is a draft PR, I'll remove the print/log statements later on. This draft PR is mostly intended to inform GSoC mentors of my progress.